packages/moonstone/internal/Picker/Picker.js

```diff
@@ -807,6 +788,12 @@ const PickerBase = class extends React.Component {
 			...rest
 		} = this.props;
 
+		const voiceProps = extractVoiceProps(rest);
+		const voiceLabelsExt = voiceProps['data-webos-voice-labels-ext'];
+		delete voiceProps['data-webos-voice-label'];
+		delete voiceProps['data-webos-voice-labels'];
+		delete voiceProps['data-webos-voice-labels-ext'];
```

Comment:
In case of Picker, specific props(ex. data-webos-voice-disabled, data-webos-voice-group) should be delegated to child buttons. so, unnecessary props is delete like above. (Delete is applied properly)

Reply:
In my opinion, Each case seems to have pros and cons. Declaring an argument value to a function can explicitly declare an item to be extracted and has a performance advantage. However, when adding an attribute value, all function call must be modified. Also there are many voice control attributes, the function declaration may become longer. If there is no big problem, I would like to maintain the current structure.